runtime/doc/ui.txt

```diff
@@ -789,19 +789,28 @@ must handle.
 	kind
 	    Name indicating the message kind:
 		"" (empty)	Unknown (consider a feature-request: |bugs|)
+		"buflist"	|:buffers| list
 		"confirm"	|confirm()| or |:confirm| dialog
 		"confirm_sub"	|:substitute| confirm dialog |:s_c|
 		"emsg"		Error (|errors|, internal error, |:throw|, …)
 		"echo"		|:echo| message
 		"echomsg"	|:echomsg| message
 		"echoerr"	|:echoerr| message
+		"highlight"	|:highlight| message
+		"inputlist"	|inputlist()| message
```

Comment:
Maybe a `_cmd` suffix is a useful convention (like `search_cmd`).

```suggestion
		"highlight_cmd"	|:highlight| message
		"inputlist"	|inputlist()| message
```

Reply:
> Not clear to me why all these things need their own special kind

Yeah they really don't. If we can come up with an appropriate umbrella kind for some of these that would be fine too. The only new kind that actually fixes an issue here is `number_prompt`; which needs to be non-fast.

Reply:
I think a `list_cmd` might suffice for buflist, highlight, let, map, set etc.